sdk/hatchery/hatchery.go

```diff
@@ -220,7 +220,19 @@ func Create(ctx context.Context, h Interface) error {
 			// Check at least one worker model can match
 			var chosenModel *sdk.Model
 			var canTakeJob bool
-			if isWithModels {
+
+			var containsRegionRequirement bool
+			for _, r := range workerRequest.requirements {
+				switch r.Type {
+				case sdk.RegionRequirement:
+					containsRegionRequirement = true
```

Comment:
add a break ;)

Reply:
@bnjjj I see you :-D

Reply:
Thanks @richardlt for taking this big responsibility 😅